File: SDKs/Aspose.Slides_Cloud_SDK_for_CSharp/src/Com/Aspose/Slides/Model/FormatSchemeResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Slides.Model {
  public class FormatSchemeResponse {
    public FormatScheme FormatScheme { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class FormatSchemeResponse {\n");
      sb.Append("  FormatScheme: ").Append(FormatScheme).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Slides-Cloud-SDK-for-Java/src/main/java/com/aspose/slides/model/FormatSchemeResponse.java
package com.aspose.slides.model;

import com.aspose.slides.model.FormatScheme;
public class FormatSchemeResponse {
  private FormatScheme FormatScheme = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getFormatScheme
	 * Gets FormatScheme
	 * @return FormatScheme
	 */
  public FormatScheme getFormatScheme() {
    return FormatScheme;
  }

	/**
	 * setFormatScheme
	 * Sets FormatScheme
	 * @param FormatScheme FormatScheme
	 */
  public void setFormatScheme(FormatScheme FormatScheme) {
    this.FormatScheme = FormatScheme;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class FormatSchemeResponse {\n");
    sb.append("  FormatScheme: ").append(FormatScheme).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

